openstack/sap-seeds/templates/flavor-seed.yaml

```diff
@@ -38,6 +38,16 @@ spec:
       "hw_video:ram_max_mb": "16"
       "catalog:alias": "m1.small,m1.smallhdd"
       "trait:CUSTOM_HANA_EXCLUSIVE_HOST": "forbidden"
+  - name: "c_c2_m2_cc3test"
+    id: "21"
+    vcpus: 2
+    ram: 2032
+    disk: 64
+    is_public: false
+    extra_specs:
+      "vmware:hv_enabled": "True"
+      "hw_video:ram_max_mb": "16"
+      "catalog:alias": "m1.xsmall_cc3test"
```

Comment:
I don't think we need an alias for it. The alias was supposed to be used for migrating away from old flavor names.

Reply:
removed alias as its new flavor